### PR TITLE
Smaller bundle size

### PR DESCRIPTION
**Pull request by timneutkens**
Redo of #219

old:

screen shot 2017-05-14 at 14 37 32

new:

screen shot 2017-05-14 at 14 38 15

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,19 +61,13 @@
...
-    "chalk": "1.1.3",
-    "clipboardy": "1.1.2",
-    "get-port": "3.1.0",
-    "ip": "1.1.5",
     "is-async-supported": "1.2.0",
     "isstream": "0.1.2",
     "media-typer": "0.3.0",
     "node-version": "1.0.0",
     "raw-body": "2.2.0",
-    "update-notifier": "2.1.0"
```
Comment (albinekb) [inline package.json:77]: no more updates 🤔

Comment (Kikobeats) [reply package.json:77]: probably better create `micro-cli` and move inside

Comment (albinekb) [reply package.json:77]: very true 👌

**Comment by Kikobeats** [comment]
looks better in terms of size, but you are removing the UX using micro from terminal, right?

File: lib/index.js [hunk 2]
```diff
@@ -1,41 +1,33 @@
...
+      if (err) {
+        console.error('micro:', err.stack)
+
+        // eslint-disable-next-line unicorn/no-process-exit
+        process.exit(1)
+      }
+
+      return listening(server, host, flags.silent)
+    })
+    .on('error', err => {
```
Comment (albinekb) [inline lib/index.js:23]: Should this be before `.listen`? 🤔

File: package.json (hunk 1 above)
Comment (timneutkens) [reply package.json:77]: `micro-dev` is going to include it for both.

**Comment by iamstarkov** [comment]
@timneutkens what is the reasoning behind this PR?

**Comment by leo** [comment]
I'm closing this PR now, since we've began working on a huge rewrite of `micro` internally. However, this does not mean that this was done for nothing. Like I discussed it with @timneutkens, we'll be taking advantage of the pre-thinking he did and also credit him for that.

Please keep an eye on the releases page! 😊 🚀

**Comment by styfle** [comment]
@iamstarkov This might be related to #234

**Comment by iamstarkov** [comment]
@styfle thanks

**Comment by iamstarkov** [comment]
@leo can you create an issue for rewrite, so we can track the progress and dont bother you in the meantime

**Comment by leo** [comment]
This won't take long - the new `micro` will have the same API.

**Pull request closed without merging**